src/components/CaptureMatching/CaptureMatchingView.js

```diff
@@ -424,6 +424,13 @@ function CaptureMatchingView() {
     [captureImage?.latitude, captureImage?.longitude]
   );
 
+  const getGroverRegDate = () => {
+    return growerAccount.first_registration_at
+      ? 'Joined at ' +
+          format(new Date(growerAccount.first_registration_at), 'MM/dd/yyyy')
```

Comment:
Can we update this to use `getDateTimeStringLocale()` so we use the appropriate format for the user?

Reply:
I think so